frontend/device/pocketbook/device.lua

```diff
@@ -342,6 +342,7 @@ end
 
 function PocketBook:initNetworkManager(NetworkMgr)
     function NetworkMgr:turnOnWifi(complete_callback)
+        inkview.WiFiPower(1)
```

Comment:
And it's harmless when the power is already active?

Reply:
That’s not something I know, unfortunately; I have not checked the documentation (is any available?)

Reply:
You probably could test by duplicating the line?

Reply:
Probably… I can test myself tomorrow or on Sunday, or we can ask @jakubjezek001 or @BloodyStupidJohnson to try.